pkgs/stdenv/darwin/default.nix

```diff
@@ -272,13 +276,18 @@ in rec {
     persistent = self: super: with prevStage; {
       inherit
         gnumake gzip gnused bzip2 gawk ed xz patch bash
-        libcxxabi libcxx ncurses libffi zlib gmp pcre gnugrep
+        ncurses libffi zlib gmp pcre gnugrep
         coreutils findutils diffutils patchutils;
 
-       llvmPackages_5 = super.llvmPackages_5 // {
-         llvm = llvmPackages_5.llvm.override { inherit libcxxabi; };
-         clang-unwrapped = llvmPackages_5.clang-unwrapped.override { llvm = self.llvmPackages_5.llvm; };
-       };
+      llvmPackages_5 = super.llvmPackages_5 // (let
+        tools = super.llvmPackages_5.tools.extend (_: _: {
+          llvm = llvmPackages_5.llvm.override { inherit libcxxabi; };
+          clang-unwrapped = llvmPackages_5.clang-unwrapped.override { llvm = self.llvmPackages_5.llvm; };
+        });
+        libraries = super.llvmPackages_5.libraries.extend (_: _: {
+          inherit (llvmPackages_5) libcxx libcxxabi;
+        });
+      in { inherit tools libraries; } // tools // libraries);
```

Comment:
@matthewbauer at some point lets make the `// tools //libraries` happen in `aliases.nix` so the overrides here can be simpler.

Reply:
Yeah, or just deprecate the versioned aliases and only expose attributes for the current version.

Reply:
Good idea!